internal/pkg/log/log.go

```diff
@@ -53,6 +53,13 @@ type baseLogger interface {
 }
 
 type contextLogger interface {
+	// Debug, Info, Warn and Error accept a template string which can contain placeholders to be replaced by ctxattr attributes.
+	// For example if the template contains "%status%" then this placeholder is replaced by an attribute with the key "status".
+	Debug(ctx context.Context, template string)
+	Info(ctx context.Context, template string)
+	Warn(ctx context.Context, template string)
+	Error(ctx context.Context, template string)
+
```

Comment:
Each method should have its own comment in the format: <methodName> ...., it doesn't matter if it is duplicated.

`Debug logs message in the debug level, you can use an attribute %placeholder% defined by the ctxattr package.`

Maybe I would choose the variable name `message` and not `template`, but I'll leave that up to you.

Reply:
Thanks.